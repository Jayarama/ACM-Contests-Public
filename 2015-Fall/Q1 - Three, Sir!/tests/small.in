15 5
5 105
11 12
14 17
135 2
