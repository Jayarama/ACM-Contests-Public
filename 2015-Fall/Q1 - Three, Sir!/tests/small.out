16
106
23
31
135
